<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleDeviceInfo</class>
 <widget class="QWidget" name="SingleDeviceInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>549</x>
     <y>180</y>
     <width>230</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>临时关闭时间:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdittmpclosetime"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdittmpstarttime"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>临时开启时间:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelworkset">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>140</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>工作参数设置:</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>180</y>
     <width>501</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>开启时间:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditstarttime"/>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>49</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>下限电压(/V):</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="lineEditlowvoltage"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>关闭时间:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditclosetime"/>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>49</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>上报间隔(/Min):</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="lineEditinterval"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>40</y>
     <width>761</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="4">
     <widget class="QLineEdit" name="lineEditIP"/>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>52</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="5">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>52</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="7">
     <widget class="QLineEdit" name="lineEditPORT"/>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>端口号:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditID"/>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="labelIP">
      <property name="text">
       <string>IP地址:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labeldeviceid">
      <property name="text">
       <string>设备ID:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line1">
   <property name="geometry">
    <rect>
     <x>-14</x>
     <y>100</y>
     <width>821</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line2">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>380</y>
     <width>821</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>330</y>
     <width>491</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QPushButton" name="Buttonmodify">
      <property name="text">
       <string>更改配置</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>82</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="Buttonrestart">
      <property name="text">
       <string>重启设备</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>82</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="Buttonclose">
      <property name="text">
       <string>关闭设备</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labeltmpset">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>临时关闭设置:</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>410</y>
     <width>391</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QPushButton" name="Buttonhistoryrecord">
      <property name="text">
       <string>历史记录查询</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>166</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="Buttonback">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
